src/test/java/net/rcarz/jiraclient/tempo/TeamTest.java

```diff
@@ -0,0 +1,158 @@
+package net.rcarz.jiraclient.tempo;
+
+import net.rcarz.jiraclient.JiraException;
+import net.rcarz.jiraclient.RestClient;
+import net.rcarz.jiraclient.RestException;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.junit.Test;
+import org.powermock.api.mockito.PowerMockito;
+
+import java.util.List;
+
+import static junit.framework.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+public class TeamTest {
+
+    @Test
+    public void testInitTeam() {
```

Comment:
But in general worth to read
https://dzone.com/articles/7-popular-unit-test-naming
my personal preference is style number 4 since it's about logic instead of implementation
However since it's not our library better to stick to initial convention, so just read and understand, no need to fix